test/e2e_node/remote/node_conformance.go

```diff
@@ -135,6 +135,7 @@ func (c *ConformanceRemote) SetupTestPackage(tardir, systemSpecName string) erro
 
 // loadConformanceImage loads node conformance image from tar file.
 func loadConformanceImage(host, workspace string) error {
+	klog.Info("loading conformance image from tarfile")
```

Comment:
can you upper case the first letter, just to keep the standard of these logs.

Reply:
yup.